sdk/keyvault/keyvault/src/az_keyvault_client.c

```diff
@@ -248,3 +237,42 @@ AZ_NODISCARD az_result az_keyvault_keys_key_get(
   // start pipeline
   return az_http_pipeline_process(&client->pipeline, &hrb, response);
 }
+
+AZ_NODISCARD az_result az_keyvault_keys_key_delete(
+    az_keyvault_keys_client * client,
+    az_span const key_name,
+    az_http_response const * const response) {
+  // Request buffer
+  uint8_t request_buffer[1024 * 4];
+  az_mut_span request_buffer_span = AZ_SPAN_FROM_ARRAY(request_buffer);
+
+  // Allocate buffer in stack to create URL like:
+  //  {vaultBaseUrl}/keys/{key-name}
+  uint8_t url_buffer[MAX_URL_SIZE];
```

Comment:
Can we get rid of this stack buffer and use the new function that appends paths to URLs?

Reply:
yes, but that is part of another PR..  so, on this one that feature is not yet enabled.
It will be converted before pushing to master and depending witch PR comes first :)

Reply:
OK, no problem